snippets/lvms-creating-lvmcluster.adoc

```diff
@@ -0,0 +1,40 @@
+:_mod-docs-content-type: SNIPPET
```

Comment:
Is there a reason why you pulled this out into a snippet? It looks like it's only included one place. The point of snippets is to have a piece of content that is going to be included several places. If that's not the case, then I'd remove this as a snippet and instead include it right in the module

Reply:
There is another proc where the full LVMCluster CR sample must be provided. I am updating that proc in another PR.

Reply:
Okay sounds good!